mkt/settings.py

```diff
@@ -381,7 +381,7 @@
 # The verification URL, the addon id will be appended to this. This will
 # have to be altered to the right domain for each server, eg:
 # https://receiptcheck.addons.mozilla.org/verify/
-WEBAPPS_RECEIPT_URL = '/verify/'
+WEBAPPS_RECEIPT_URL = 'http://mp.dev/receipt-verifier/'
```

Comment:
Oh, hmm, shouldn't this default be changed from ``localhost`` to ``mp.dev``? https://github.com/andymckay/zamboni/blob/1109334/mkt/settings.py#L104 I think Docker sets it anyway via the environment.